repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
The separate repo is how Universe packages install CLI tools. See how spark does it. If you remember from around a month ago, we had a push to 'make hdfs configurable' so I implemented the only solution that was within the time constraints. Since we have another three weeks now, we can add a better solution to the roadmap.